Automattic/studio
# Studio: add affordance to expand all the initial prompts

Closes: https://github.com/Automattic/dotcom-forge/issues/8982

## Proposed Changes

This PR adds `More suggestions` button to open more prompts to the AI Assistant tab in Studio.

## Testing Instructions



* Pull the changes from this PR
* Start the app with `nvm use && npm install && npm start`
* Navigate to `AI Assistant` tab in Studio for a specific site
* Observe that you are presented with welcome prompts
* Observe that you can see `More suggestions` button right below the third welcome prompt
* Click on the button
* Confirm that you can see more prompts
* Confirm that clicking the button scrolls to the bottom of the last prompt   

## Pre-merge Checklist
